Assets/Resources/X/Brushes/DryBrush/DryBrush.asset

```diff
@@ -14,7 +14,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   m_Guid:
     m_storage: 725f4c6a-6427-6524-29ab-da371924adab
-  m_DurableName: 
+  m_DurableName: DryBrush
```

Comment:
We have a few brushes that still don't have `m_DurableName` set. Is this deliberate?

```
Assets/Resources/Brushes/Basic/Felt/Felt.asset:  m_DurableName:
Assets/Resources/Brushes/Basic/TaperedHighlighter/TaperedHighlighter.asset:  m_DurableName:
Assets/Resources/Brushes/New BrushDescriptor.asset:  m_DurableName:
Assets/Resources/X/Brushes/Test/TubeDiffuse.asset:  m_DurableName:
Assets/Resources/X/Brushes/Test/TubeUnlit.asset:  m_DurableName:
```

Reply:
Not sure. I think my strategy was "export > fix > repeat" and it's possible the test scene I had wasn't using all brushes. I wonder if those brushes aren't user-accessible? That would be the reason they would be missing from the test scene.

Reply:
I'll fix 'em anyway as they can be accessed via the API

Reply:
gotcha!